CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 Fix:  Fixed the bug about recovering ONTIMEINTERVAL subscriptions (Issue #693)
+Fix:  Fixed the crash in queryContext with a double geoscope (Issue #690)
```

Comment:
I'd suggest:

"Fix: Fixed the crash in queryContext due to polygon external shell situations with geoscope (Issue #690)"